src/locales/pt-br.ts

```diff
@@ -50,60 +50,69 @@ export default {
+    mary_mother_of_god: 'Maria, Mãe de Deus',
+    palm_sunday: 'Domingo de Ramos',
+    pentecost_sunday: 'Pentecostes',
+    peter_and_paul_apostles: 'São Pedro e São Paulo, Apóstolos',
+    presentation_of_the_lord: 'Apresentação do Senhor',
+    most_sacred_heart_of_jesus: 'Sagrado Coração de Jesus',
+    exaltation_of_the_holy_cross: 'Exaltação da Santa Cruz',
     transfiguration: 'Transfiguração do Senhor',
-    trinitySunday: 'Santíssima Trindade',
+    trinity_sunday: 'Santíssima Trindade',
   },
   sanctoral: {
-    ourLadyHelpOfChristians: 'Nossa Senhora, Auxílio dos Cristãos',
-    ourLadyMediatrix: 'Nossa Senhora, Medianeira',
-    ourLadyMediatrixOfAllGrace: 'Nossa Senhora, medianeira de todas as Graças',
-    ourLadyMotherOfDivineProvidencePatronessOfPuertoRico: 'Nossa Senhora, Mãe da Divina Providência',
-    ourLadyMotherOfMercy: 'Nossa Senhora, Mãe de Misericórdia',
-    ourLadyOfAparecidaPatronessOfBrazil: 'Nossa Senhora Aparecida',
-    ourLadyOfChina: 'Nossa Senhora da China',
-    ourLadyOfFatima: 'Nossa Senhora de Fatima',
-    ourLadyOfGoodCounsel: 'Nossa Senhora do Bom Conselho',
-    ourLadyOfGuadalupe: 'Nossa Senhora de Guadalupe',
-    ourLadyOfHungary: 'Nossa Senhora da Hungria',
-    ourLadyOfLoreto: 'Nossa Senhora de Loreto',
-    ourLadyOfLourdes: 'Nossa Senhora de Lourdes',
-    ourLadyOfLujanPatronessOfArgentina: 'Nossa Senhora de Luján, padroeira da Argentina',
-    ourLadyOfMountCarmel: 'Nossa Senhora do Monte Carmelo',
-    ourLadyOfPeace: 'Nossa Senhora da Paz',
-    ourLadyOfPerpetualHelp: 'Nossa Senhora do Perpétuo Socorro',
-    ourLadyOfThePillar: 'Nossa Senhora do Pilar',
-    ourLadyOfTheRosary: 'Nossa Senhora do Rosário',
-    ourLadyQueenOfPeace: 'Nossa Senhora, Rainha da Paz',
-    presentationOfTheBlessedVirginMary: 'Apresentação da Virgem Maria',
-    saintJohnPaulIiPope: 'Papa São João Paulo II',
-    saintJohnXxiiiPope: 'Papa São João XXIII',
+    our_lady_help_of_christians: 'Nossa Senhora, Auxílio dos Cristãos',
+    our_lady_mediatrix_of_all_grace: 'Nossa Senhora, Medianeira de todas as Graças',
+    our_lady_mother_of_divine_providence_principal_patroness_of_puerto_rico: 'Nossa Senhora, Mãe da Divina Providência',  // TODO: Add `Principal Patronness of Porto Rico` title
+    our_lady_mother_of_mercy: 'Nossa Senhora, Mãe de Misericórdia',
+    our_lady_of_aparecida_principal_patroness_of_brazil: 'Nossa Senhora Aparecida',  // TODO: Add `Principal Patronness of Brazil` title
+    our_lady_of_china: 'Nossa Senhora da China',
+    our_lady_of_fatima: 'Nossa Senhora de Fatima',
+    our_lady_of_good_counsel: 'Nossa Senhora do Bom Conselho',
+    our_lady_of_guadalupe: 'Nossa Senhora de Guadalupe',
+    our_lady_of_hungary_principal_patroness_of_hungary: 'Nossa Senhora da Hungria',
+    our_lady_of_loreto: 'Nossa Senhora de Loreto',
+    our_lady_of_lourdes: 'Nossa Senhora de Lourdes',
+    our_lady_of_lujan_principal_patroness_of_argentina: 'Nossa Senhora de Luján, padroeira da Argentina',
```

Comment:
@HenriqueDerosa, could you add the Brazilian Portuguese patronage titles please? Below I list them all. There are also some other issue which you might be able to fix. Thanks in advance. :smiley:

```js
our_lady_mother_of_divine_providence_principal_patroness_of_puerto_rico: 'Nossa Senhora, Mãe da Divina Providência',  // TODO: Add `Principal Patronness of Porto Rico` title
our_lady_of_aparecida_principal_patroness_of_brazil: 'Nossa Senhora Aparecida',  // TODO: Add `Principal Patronness of Brazil` title
our_lady_of_hungary_principal_patroness_of_hungary: 'Nossa Senhora da Hungria',  // TODO: Add `Principal Patronness of Hungary` title
our_lady_of_lujan_principal_patroness_of_argentina: 'Nossa Senhora de Luján, padroeira da Argentina',  // TODO: Add `Principal` word before `padroeira`
```

Reply:
@HenriqueDerosa, ping :smiley: